src/__stories__/callout-story.tsx

```diff
@@ -55,28 +46,20 @@ export const Default: StoryComponent<Args> = ({
     ) : undefined;
 
     return (
-        <ThemeVariant isInverse={isOverInverse}>
-            <div
-                style={{
-                    background: isOverInverse ? skinVars.colors.backgroundBrand : skinVars.colors.background,
-                }}
-            >
-                <ResponsiveLayout>
-                    <Box paddingY={24}>
-                        <Callout
-                            icon={withIcon ? <IconBoxLight /> : undefined}
-                            onClose={isClosable ? () => {} : undefined}
-                            title={title}
-                            description={description}
-                            button={button}
-                            secondaryButton={secondaryButton}
-                            buttonLink={buttonLink}
-                            aria-label="Callout label"
-                        />
-                    </Box>
-                </ResponsiveLayout>
-            </div>
-        </ThemeVariant>
+        <ResponsiveLayout isInverse={isOverInverse}>
```

Comment:
same here with inverse / isOverInverse, we try to keep consistency between stories

Reply:
fixed. It was also updated in all the button's stories